src/main/java/ch/njol/skript/expressions/ExprArgument.java

```diff
@@ -81,12 +82,13 @@ public class ExprArgument extends SimpleExpression<Object> {
 	private Argument<?> argument;
 
 	private int ordinal = -1; // Available in ORDINAL and sometimes CLASSINFO
+	private boolean SUPPORTS_UNKNOWN_EVENT = Skript.classExists("org.bukkit.event.command.UnknownCommandEvent");
```

Comment:
Should change L64 to  `@Since("1.0, 2.7 (command events), INSERT VERSION (unknown command)")`